src/app/components/nav-bar/nav-bar.component.html

```diff
@@ -30,6 +30,7 @@
                     <li>
                         <a [routerLink]="['/home']" >Home</a></li>
                         <li><a [routerLink]="['/tasks']">Tasks</a></li>
+                    <li><a [routerLink]="['/profil']">Profil</a></li>
                     <li><a [routerLink]="['/teams']">About us</a></li>
```

Comment:
The first two li tags have different indentation compared to the rest of the code.